DOL.WHD.Section14c.PdfAPi/PdfHelper/Constants.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace DOL.WHD.Section14c.PdfApi.PdfHelper
+{
+    public class Constants
+    {
+        public const string PdfConcatenateSupportedFileTypes = "pdf|png|jpg|jpeg";
```

Comment:
@mgwalker aaah the vagaries of OOP :) Looks like David moved this from config to code to prevent adding unsupported types in web.config (from a later comment)